README.md

```diff
@@ -1,175 +1,404 @@
+- [Wren MCP Server](https://github.com/Canner/wren-engine). Provides a semantic engine powering business intelligence for Postgres and other databases.
 
-```bash
-docker run -it --rm crystaldba/postgres-mcp -h hostname -p 5432 -U username -d dbname
-```
+DBA Tools (including commercial offerings)
+- [Aiven Database Optimizer](https://aiven.io/solutions/aiven-ai-database-optimizer). A tool that provides holistic database workload analysis, query optimizations, and other performance improvements.
+- [dba.ai](https://www.dba.ai/). An AI-powered database administration assistant that integrates with GitHub to resolve code issues.
+- [pgAnalyze](https://pganalyze.com/). A comprehensive monitoring and analytics platform for identifying performance bottlenecks, optimizing queries, and real-time alerting.
+- [Postgres.ai](https://postgres.ai/). An interactive chat experience combining an extensive Postgres knowledge base and GPT-4.
+- [Xata Agent](https://github.com/xataio/agent). An open-source AI agent that automatically monitors database health, diagnoses issues, and provides recommendations using LLM-powered reasoning and playbooks.
+
+Postgres Utilities
+- [Dexter](https://github.com/DexterDB/dexter). A tool for generating and testing hypothetical indexes on PostgreSQL.
+- [PgHero](https://github.com/ankane/pghero). A performance dashboard for Postgres, with recommendations.
+Postgres Pro incorporates health checks from PgHero.
+- [PgTune](https://github.com/le0pard/pgtune?tab=readme-ov-file). Heuristics for tuning Postgres configuration.
+
+## Frequently Asked Questions
+
+*How is Postgres Pro different from other Postgres MCP servers?*
+There are many MCP servers allow an AI agent to run queries against a Postgres database.
+Postgres Pro does that too, but also adds tools for understanding and improving the performance of your Postgres database.
+For example, it implements a version of the [Anytime Algorithm of Database Tuning Advisor for Microsoft SQL Server](https://www.microsoft.com/en-us/research/wp-content/uploads/2020/06/Anytime-Algorithm-of-Database-Tuning-Advisor-for-Microsoft-SQL-Server.pdf), a modern industrial-strength algorithm for automatic index tuning.
+
+*Why are MCP tools needed when the LLM can reason, generate SQL, etc?*
+LLMs are invaluable for tasks that involve ambiguity, reasoning, or natural language.
+When compared to procedural code, however, they can be slow, expensive, non-deterministic, and sometimes produce unreliable results.
+In the case of database tuning, we have well established algorithms, developed over decades, that are proven to work.
+Postgres Pro lets you combine the best of both worlds by pairing LLMs with classical optimization algorithms and other procedural tools.
+
+*How do you test Postgres Pro?*
+Testing is critical to ensuring that Postgres Pro is reliable and accurate.
```

Comment:
We don't need this sentence -- it feels like fluff. The FAQ question is a hard hitting question, so answer it instantly and boldly in the next words.